scripts/check-sample-lnd-conf.sh

```diff
@@ -14,33 +14,35 @@ CONF_FILE=${2:-sample-lnd.conf}
 # We are reading the default values of lnd from lnd --help. To avoid formatting
 # issues the width of the terminal is set to 240. This needs a workaround for
 # CI where we don't have an interactive terminal.
+FILE_TMP=$(mktemp)
 if [ -t 0 ]; then
     stty cols 240
-    LND_HELP="$(go run -tags="$TAGS" \
-        github.com/lightningnetwork/lnd/cmd/lnd --help)"" --end"
+    go run -tags="$TAGS" github.com/lightningnetwork/lnd/cmd/lnd --help > \
+        $FILE_TMP
 else
     tmux new-session -d -s simulated-terminal -x 240 -y 9999
     tmux send-keys -t simulated-terminal.0 "go run -tags=\"$TAGS\" \
-        github.com/lightningnetwork/lnd/cmd/lnd --help; tmux wait -S run" ENTER
+        github.com/lightningnetwork/lnd/cmd/lnd --help >"$FILE_TMP"; \
+        tmux wait -S run" ENTER
     tmux wait-for run
-    LND_HELP="$(tmux capture-pane -t simulated-terminal.0 -p)"" --end"
     tmux kill-session -t simulated-terminal
 fi
 
+LND_HELP="$(cat $FILE_TMP)"" --end"
+
 # LND_OPTIONS is a list of all options of lnd including the equal sign, which
 # is needed to distinguish between booleans and other variables.
 # It is created by reading the first two columns of lnd --help. 
-LND_OPTIONS="$(go run -tags="$TAGS" \
-    github.com/lightningnetwork/lnd/cmd/lnd --help | \
+LND_OPTIONS="$(cat $FILE_TMP | \
     awk '{
         option=""; 
         if ($1 ~ /^--/){option=$1};
         if ($2 ~ /^--/){option=$2}; 
-        if (match(option,  /--[^=]+[=]*/) && 
-                substr(option, length(option)) != "-")
+        if (match(option,  /--[^=]+[=]*/))
```

Comment:
I guess I don't really understand `awk` after all. What's the significance of this change?

Reply:
That was just a (dirty) workaround. The awk script checks whether the first columns represent a valid option name, in order to avoid having text fragments with -- from the descriptions being recognized as an option. If an option is recognized, only the text up to the equals sign is returned.
In a narrow window, it could happen that the option labels from the descriptions were given line breaks and were displayed as the first column (e.g. --tlxextra-). With the clean implementation now, the workaround is no longer needed.